package.json

```diff
@@ -34,7 +34,7 @@
     "react-spinners": "^0.11.0",
     "react-toastify": "^8.1.0",
     "recharts": "^2.1.6",
-    "set.js": "^0.4.1",
+    "set.js": "^1.0.0",
```

Comment:
lol does this even exist? I don't see 1.0.0 in 
- https://github.com/SetProtocol/set.js/releases
- https://github.com/SetProtocol/set.js/tags